docs/jwst/straylight/reference_files.rst

```diff
@@ -1,11 +1,7 @@
 Reference Files
 ===============
-The ``straylight`` step uses the :ref:`REGIONS <regions_reffile>` reference
-file, which stores locations of the slice regions on the detectors. This
-reference file provides 2-D detector images in which each pixel is set to
-the number of the corresponding slice (or 0 for inter-slice pixels) at each of 9
-different throughput levels ranging from 10% - 90%.  While ``assign_wcs``
-uses a fairly exclusive slice mask (selecting only pixels with high
-throughput for science analysis), the ``straylight`` step uses a very
-inclusive 20% threshold to define slice pixels and obtain a cleaner
-sample of inter-slice pixels.
+The ``straylight`` step uses the :ref:`MRSXARTCORR <mrsxartcorr_reffile>` reference
+file, which stores vectors describing the appropriate cross-artifact convolution kernel
+for each MRS band.  In Channel 1 these vectors include power in a broad Lorentzian core
+plus a pair of double-Gaussian profiles.  In Channels 2 and 3 these vectors include only
+power in the broad Lorentzian, while in Channel 4 there is no correction.
```

Comment:
The "mrsxartcorr_reffile.inc" file (defined above) is not getting included anywhere from any of the other .rst files, hence it's not getting compiled at all and not included in the doc build. Typically that .inc file is loaded somewhere in the "reference_files.rst" doc that describes the reference file. So just add a line here at the end of this file:
`.. include:: ../references_general/mrsxartcorr_reffile.inc`

If you want, you could even move this little paragraph of text to the top of the mrsxartcorr_reffile.inc and reduce this file to only contain:
```
Reference Files
===========
The ``straylight`` step uses a MRSXARTCORR reference file.

.. include:: ../references_general/mrsxartcorr_reffile.inc
```